plugins/legacy-plugin-chart-sunburst/src/index.js

```diff
@@ -20,6 +20,7 @@ import { t } from '@superset-ui/translation';
 import { ChartMetadata, ChartPlugin } from '@superset-ui/chart';
 import transformProps from './transformProps';
 import thumbnail from './images/thumbnail.png';
+import controlPanel from './controlPanel.ts';
```

Comment:
please remove `.ts` This may break in consumer app because it may refer to the `.js` transpiled file.

Reply:
Hmm... originally I didn't have the .ts there, but CI was failing due to a linting error about it.

Reply:
removed, and eslint-disable'd

Reply:
hmm. we should consider changing that to warning or sth. thanks for the note.